lib/src/command/logout.dart

```diff
@@ -16,12 +16,18 @@ class LogoutCommand extends PubCommand {
   @override
   String get description => 'Log out of pub.dev.';
   @override
-  bool get takesArguments => false;
+  bool get takesArguments => true;
 
   LogoutCommand();
 
   @override
   Future<void> runProtected() async {
-    oauth2.logout(cache);
+    if (argResults.rest.isEmpty) {
+      oauth2.logout(cache);
+    } else if (argResults.rest.length > 1) {
+      usageException('Takes only a single argument.');
```

Comment:
> It's also possible we should consider making it dart pub login/logout --server <server> just because it might read better in the CLI documentation when probing with --help.

Done ✅